DOC-8266: Correct Datadog metric export docs (step 2) (#17534)
cockroachdb/docs: src/current/cockroachcloud/export-metrics.md

<section class="filter-content" markdown="1" data-scope="aws-metrics-export">

Exporting metrics to AWS CloudWatch is only available on {{ site.data.products.dedicated }} clusters which are hosted on AWS. If your {{ site.data.products.dedicated }} cluster is hosted on GCP, you can [export metrics to Datadog](export-metrics.html?filters=datadog-metrics-export) instead.

> {{site.data.alerts.callout_info}}
> Enabling metrics export will send around 250 metrics per node to AWS CloudWatch.
> 
> Consult with [AWS CloudWatch documentation](https://aws.amazon.com/cloudwatch/pricing/) to gauge how this adds to your AWS CloudWatch spend.

Reviewer: ```suggestion
{{site.data.alerts.callout_info}}
Enabling metrics export will send around 250 metrics per node to AWS CloudWatch. Review the [AWS CloudWatch documentation](https://aws.amazon.com/cloudwatch/pricing/) to gauge how this adds to your AWS CloudWatch spend.
```

Is this the same edit as https://github.com/cockroachdb/docs/pull/17438/files?

Reviewer: If so, you can probably close the other PR and merge the edits in this PR
